org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -38,6 +38,13 @@ <h3>Fixed Bugs</h3>
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/672">#672</a>).</li>
 </ul>
 
+<h3>Non-functional Changes</h3>
+<ul>
+  <li>Improved error message when already instrumented classes are used for
+      instrumentation or analysis.
```

Comment:
Existing items consisting of one sentence use only one dot after link to GitHub.